content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md

```diff
@@ -0,0 +1,151 @@
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+Atualiza o cluster Kubernetes para uma versão específica
+
+### Sinopse
+
+Atualiza o cluster Kubernetes para uma versão específica
+
+```
+kubeadm upgrade apply [versão]
+```
+
+### Opções
+
+<table style="width: 100%; table-layout: fixed;">
+<colgroup>
+<col span="1" style="width: 10px;" />
+<col span="1" />
+</colgroup>
+<tbody>
+
+<tr>
+<td colspan="2">--allow-experimental-upgrades</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Exibe as versões instáveis do Kubernetes como uma alternativa de atualização e permite a atualização para versões candidatas alfa/beta/release do Kubernetes.</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--allow-release-candidate-upgrades</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Exibe as versões candidatas a lançamento do Kubernetes como uma alternativa de atualização e permite a atualização para versões candidatas a lançamento do Kubernetes.</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--certificate-renewal&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Padrão: true</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Executa a renovação dos certificados usados pelo componente alterado durante as atualizações.</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--config string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Caminho para um arquivo de configuração do kubeadm.</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--dry-run</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Não aplica as modificações; apenas exibe as alterações que seriam efetuadas.</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--etcd-upgrade&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Padrão: true</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Atualiza o etcd.</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--feature-gates string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Um conjunto de pares chave=valor que descreve feature gates para várias funcionalidades. As opções são:<br/>
+PublicKeysECDSA=true|false (ALPHA - padrão=false)<br/>RootlessControlPlane=true|false (ALPHA - padrão=false)<br/>UnversionedKubeletConfigMap=true|false (padrão=true)
+</p></td>
+</tr>
+
+<tr>
+<td colspan="2">-f, --force</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Força a atualização, embora alguns requisitos possam não ser atendidos. Isso também implica o modo não interativo.</p></td>
```

Comment:
Suggestion: "possam não estar sendo atendidos" instead of "possam não ser". I feel the current wording kind of gives the impression that it's a definitive state, rather than a changeable one.